### ISSUE TITLE
Byzantium with two ref. beliefs

### ISSUE THREAD

**Issue by Der-Zorn-Gottes** [LoneGazebo/Community-Patch-DLL#1600]
In my last game, I conquered Constantinople, and noticed that I could build two religious wonders. Is it possible for Byzanz to have two reformation beliefs?
This game is already done and with another version, so I can' go back and check weather they had the basilika. But would their UA allow it in principle to reform twice, once from 20% and once from basilika?

**Comment by LoneGazebo** [comment]
No, only once. It's a bool that flips once you've done it.

**Comment by Der-Zorn-Gottes** [comment]
Here is the picture of that city:
![20160407191500_1](https://cloud.githubusercontent.com/assets/16065630/14572286/3a98e9e6-034e-11e6-9451-bc5cb392c0b4.jpg)

**Comment by LoneGazebo** [comment]
Yeah, you can build the building twice, but the building simply flips a boolean switch to true - if the religion has an open reformation slot, you'd get to pick one, otherwise it is just a national wonder with some buffs.
